app/javascript/components/App.js

```diff
@@ -12,14 +14,33 @@ import {
 } from "react-router-dom"
 
 class App extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      apartments: []
+    }
+  }
+
+  componentDidMount() {
+    this.apptRead()
+  }
+
+  apptRead = () => {
+    fetch("http://localhost:3000/apartments")
```

Comment:
ooo this is fun, instead of 
```javascript
fetch("http://localhost:3000/apartments")
```
try 
  
```javascript
fetch("/apartments")
```
Let me know what you get